fake.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package testing
+
+import (
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+)
+
+// FakeCallArgs holds all the args passed in a call.
+type FakeCallArgs map[string]interface{}
```

Comment:
Yeah, I see what you mean.  It definitely simplifies things if you don't worry about the parameter names.